boards/arm/olimex_stm32_h103/olimex_stm32_h103.yaml

```diff
@@ -0,0 +1,23 @@
+identifier: olimex_stm32_h103
+name: OLIMEX-STM32-H103
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 20
+flash: 128
+supported:
+  - adc
+  - gpio
+  - i2c
+  - pinmux
```

Comment:
pinmux could be removed as there is no associated 'pinmux' test (unlike other peripherals listed as supported here

Reply:
There are other peripherals supported by the MCU (like DMA, or CAN), but I have not tested that, should they be included too?

Reply:
You can add them later, once tested.